scanner/src/main/kotlin/scanners/fossid/FossId.kt

```diff
@@ -442,6 +442,9 @@ class FossId internal constructor(
         projectCode: String,
         projectName: String
     ): String {
+        if (url.contains("@")) {
```

Comment:
Use Kotlin style: `"@" in url`

Reply:
Also, could we simply use `String.isValidUri()` or even ` String.replaceCredentialsInUri()` here to strip credentials?

Reply:
I wanted to warn the user that it should not give credentials, because FossID scanner will add its own.

isValidUri is too generic, it won't allow me to display a specific warning message to this case I want to address.

Same for replaceCredentialsInUri: It will silently digest this case, leaving a URL that could be not valid without the given credentials, and won't output a warning making this case hard to debug.